commit 05666e70f1ef63632aea8b1aed84351a6e30d0d5
Author: Jan Tojnar <jtojnar@gmail.com>
Date:   Thu Jul 23 18:49:03 2020 +0200

    Improve separation between malcontent and malcontent-ui packages
    
    These artefacts go into both packages by default:
    
    * malcontent-control – a CLI tool
    * PAM module
    * gettext locale data
    * help pages ✨
    * AccountService/D-Bus interfaces
    * Polkit com.endlessm.ParentalControls rules and policy
    
    Items marked with ✨ will go to ui package, while the rest is installed with the base library.
    
    See also the packaging done by upstream in Endless:
    https://github.com/endlessm/malcontent/tree/5482b7eb99aee9b8c344629bc078e193d3c26ffc/debian
    
    Cannot upstream this since they are worried about maintainability:
    https://gitlab.freedesktop.org/pwithnall/malcontent/merge_requests/38#note_437946

diff --git a/meson.build b/meson.build
index 038dd35..e3c77f0 100644
--- a/meson.build
+++ b/meson.build
@@ -123,9 +123,8 @@ test_env = [
   'LC_ALL=C.UTF-8',
 ]
 
-subdir('accounts-service')
-subdir('help')
 if not get_option('use_system_libmalcontent')
+  subdir('accounts-service')
   subdir('libmalcontent')
 else
   libmalcontent_api_version = '0'
@@ -143,14 +142,19 @@ if get_option('ui').enabled()
   )
   subdir('libmalcontent-ui')
 endif
-subdir('malcontent-client')
+if not get_option('use_system_libmalcontent')
+  subdir('malcontent-client')
+endif
 if get_option('ui').enabled()
   subdir('malcontent-control')
+  subdir('help')
 
   gnome.post_install(
     gtk_update_icon_cache: true,
     update_desktop_database: true,
   )
 endif
-subdir('pam')
-subdir('po')
+if not get_option('use_system_libmalcontent')
+  subdir('pam')
+  subdir('po')
+endif
